local-volume/provisioner/pkg/common/common.go

```diff
@@ -199,12 +211,17 @@ func VolumeConfigToConfigMapData(config *ProvisionerConfiguration) (map[string]s
 	}
 	configMapData[ProvisonerStorageClassConfig] = string(val)
 	if len(config.NodeLabelsForPV) > 0 {
-		nodeLabels, err := yaml.Marshal(config.NodeLabelsForPV)
-		if err != nil {
-			return nil, fmt.Errorf("unable to Marshal node label: %v", err)
+		nodeLabels, nlErr := yaml.Marshal(config.NodeLabelsForPV)
+		if nlErr != nil {
+			return nil, fmt.Errorf("unable to Marshal node label: %v", nlErr)
 		}
 		configMapData[ProvisionerNodeLabelsForPV] = string(nodeLabels)
 	}
+	ver, err := yaml.Marshal(config.UseAlphaAPI)
```

Comment:
What happens of configmap is missing the entry? Will it default to false? Can you add a unit test for this?

Reply:
Wanted to add unit test for `LoadProvisionerConfigs` to test the default value of `UseAlphaAPI` ( which will not execute `Marshal` functions)
But when i made dir for `/etc/provisioner/config/` which is the `ProvisionerConfigPath`, `permission denied` will occur, any thoughts ?

test like this:
```
func TestLoadProvisionerConfigs(t *testing.T) {
       if err := os.MkdirAll("/etc/provisioner/config/", 0777); err != nil {
               t.Fatal("create config dir error: ", err)
       }
       provisionerConfig := &ProvisionerConfiguration{
               StorageClassConfig: make(map[string]MountConfig),
       }
       LoadProvisionerConfigs(provisionerConfig)

       if provisionerConfig.UseAlphaAPI == true {
               t.Errorf("UseAlphaAPI should default to false")
       }
}
```

Reply:
You could add provisioner config as a variable to the function.  Then in the unit test, it could be a tmpdir, and in production it could be /etc/provisioner/config.

Reply:
done, PTAL